src/main/java/org/broadinstitute/consent/http/resources/ResearcherResource.java

```diff
@@ -113,14 +39,4 @@ public Response deleteAllProperties(@PathParam("userId") Integer userId) {
             return createExceptionResponse(e);
         }
     }
-
-    private User findByAuthUser(AuthUser user) {
-        GoogleUser googleUser = user.getGoogleUser();
-        User dacUser = userService.findUserByEmail(googleUser.getEmail());
```

Comment:
As a result of removing these methods,`UserService` and `LibraryCardService` are now unused

Reply:
I have to leave them in the constructor because of injection, but I've removed them as variables.

Reply:
You can remove them from the constructor. That will require additional changes in ConsentApplication and maybe the tests too.